test/tests/catalog_functions.cpp

```diff
@@ -76,6 +76,7 @@ void TestGetTypeInfo(HSTMT &hstmt, std::map<SQLSMALLINT, SQLULEN> &types_map) {
 	    SQL_DOUBLE,
 	    SQL_VARCHAR,
 	    SQL_VARBINARY,
+	    SQL_UNKNOWN_TYPE,
```

Comment:
Can you explain why you've added this?

Reply:
This test runs `SQLGetTypeInfo` with `SQL_ALL_TYPES`, which returns all types listed in `ApiInfo::ODBC_SUPPORTED_SQL_TYPES`. Sine we added a line item to that list (`"'UUID'"`) we need to add its corresponding ODBC type here as well (`0` or `SQL_UNKNOWN_TYPE`).